lib/bel/translator/plugins/rdf/translator.rb

```diff
@@ -94,6 +94,16 @@ def write_rdf_to_io(format, objects, io, options = {})
         :stream => true
       )
 
+      if (@format == :turtle)
```

Comment:
We shouldn't need to restrict these prefixes to only the `:turtle` format. Let's remove this condition so other RDF formats that support prefixes will use the YAML definitions.